src/wwwroot/js/genpage/params.js

```diff
@@ -274,18 +274,33 @@ function genInputs(delay_final = false) {
             let inputHeightParent = findParentOfClass(inputHeight, 'auto-slider-box');
             let inputHeightSlider = getRequiredElementById('input_height_rangeslider');
             let resGroupLabel = findParentOfClass(inputWidth, 'input-group').querySelector('.header-label');
+            let inputAspectRatioParent = findParentOfClass(inputAspectRatio, 'auto-dropdown-box');
+            let inputAspectRatioParentStyles = window.getComputedStyle(inputAspectRatioParent);
+            let swapAspectRatioButton = document.createElement("button");
+            inputAspectRatioParent.style.position = 'relative';
+            swapAspectRatioButton.style.display = inputAspectRatio.value == "Custom" ? 'block' : 'none';
+            swapAspectRatioButton.style.right = inputAspectRatioParentStyles.paddingRight;
+            swapAspectRatioButton.style.top = inputAspectRatioParentStyles.paddingTop;
+            // Needed to override the padding of basic-button class
+            swapAspectRatioButton.style.setProperty('padding', '0 5px', 'important');
```

Comment:
But that needs to be dynamic because the padding of the container changes between themes and the !important won't work on the css file for some ordering issue (javascript has priority so I did that there)

Reply:
gonna fist-fight modern why it gotta be full of `!important`s omg

Reply:
Thanks, I will look around for other stuff that I can implement with my knowledge :)

Reply:
er... nope. I copied it over to css and it just works. not sure what your issue was?